requirements/skinny-requirements.txt

```diff
@@ -14,3 +14,6 @@ requests<3,>=2.17.3
 packaging<24
 importlib_metadata<7,>=3.7.0,!=4.7.0
 sqlparse<1,>=0.4.0
+azure_storage_blob<13,>=12.15.0
+google-cloud-storage<3,>=2.7.0
+boto3<2,>=1.26.91
```

Comment:
Can we instead add these to MLR and MLflow core? I don't think these are critical for skinny, and Databricks users trying to use MLflow with UC from their laptops should install MLlflow core